src/Form/Form.js

```diff
@@ -228,7 +227,13 @@ class Form {
         }
 
         const handler = (e) => {
-            if (this.tooltip || this.isAutofilling) return
+            if (this.device.getActiveTooltip() || this.isAutofilling) return
+
+            const input = e.target
+            const getPosition = () => {
```

Comment:
This _getPosition_ part is kinda unfortunate but I don't think there's an obvious other pattern for calculating the position of the input without having the copy of the node.
The plan is the top autofill will pass stable values here as we don't need it; I didn't test this in my other patch and instead had it hardcoded; the rest is all the same as #49 minus the top autofill parts.